eq-publisher/src/utils/convertPipes.js

```diff
@@ -57,12 +61,18 @@ const convertElementToPipe = ($elem, ctx) => {
   if (!entity) {
     return "";
   }
+
   const output = pipeConfig.render(entity);
   const dataType = pipeConfig.getType(entity);
-
   const filter = FILTER_MAP[dataType];
+  let unitType;
 
-  return filter ? `{{ ${filter(output)} }}` : `{{ ${output} }}`;
+  if (dataType === "Unit") {
```

Comment:
Can this use the constant from `pipingAnswerTypes.js` instead if a string, please?

Reply:
thanks Matt - sorted both - makes sense!